src/main/java/de/hysky/skyblocker/mixins/BackgroundRendererMixin.java

```diff
@@ -0,0 +1,45 @@
+package de.hysky.skyblocker.mixins;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import de.hysky.skyblocker.config.SkyblockerConfigManager;
+import de.hysky.skyblocker.config.configs.CrimsonIsleConfig;
+import de.hysky.skyblocker.utils.Utils;
+import net.minecraft.block.enums.CameraSubmersionType;
+import net.minecraft.client.render.BackgroundRenderer;
+import net.minecraft.client.render.Camera;
+import net.minecraft.client.render.FogShape;
+import net.minecraft.util.math.MathHelper;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(BackgroundRenderer.class)
+public abstract class BackgroundRendererMixin {
+
+    /**
+     * Moves fog farther away from the player when in the crimson isles.
+     * This sets it to be the same distance as what you would see in the overworld (every other skyblock island)
+     */
+    @Inject(method = "applyFog", at = @At("RETURN"))
+    private static void applyFogModifyDistance(Camera camera, BackgroundRenderer.FogType fogType, float viewDistance, boolean thickFog, float tickDelta, CallbackInfo info) {
```

Comment:
should probably change `info` to `ci` to match rest of the mod. I would also recommend using `@Local(argsOnly = true)` to take in the target method' parameters as not all of them are used so taking only what we need should be more robust against future updates by Mojang to this method. Note that for the `viewDistance` float you'll need to specify an `ordinal` of 0.